package.json

```
@@ -21,6 +21,9 @@
     "fs": "0.0.1-security",
     "google-auth-library": "^6.0.5",
     "googleapis": "^55.0.0",
-    "nodemon": "^2.0.4"
+    "lodash": "^4.17.19",
```

```suggestion
    "lodash": "^4.17.19",
    "node-gyp": "^7.0.0",
    "quick.db": "^7.1.1"
```
What are these package for?